{"uid":"5ce8d0f3fc5b89af","name":"Checks that remote card button text includes counter which changed after click & check value reverted after reload","fullName":"Checks that remote card button text includes counter which changed after click & check value reverted after reload","historyId":"e8cba62d06a9c53a496a03db1427dc1e","time":{"start":1681299816036,"stop":1681299817565,"duration":1529},"status":"failed","statusMessage":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","statusTrace":"TypeError: The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application\n    at <unknown> (http://localhost:4173/src/bootstrap.tsx:5:10)","flaky":true,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":1,"retriesStatusChange":false,"beforeStages":[],"testStage":{"status":"failed","statusMessage":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","statusTrace":"TypeError: The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application\n    at <unknown> (http://localhost:4173/src/bootstrap.tsx:5:10)","steps":[{"name":"\"before each\" hook","time":{"start":1681299816036,"stop":1681299816389,"duration":353},"status":"passed","steps":[{"name":"visit http://localhost:4173","time":{"start":1681299816131,"stop":1681299816388,"duration":257},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"http://localhost:4173 request","time":{"start":1681299816388,"stop":1681299816388,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"http://localhost:4173 request","time":{"start":1681299816388,"stop":1681299816388,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"attachments":[],"parameters":[],"stepsCount":3,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":true},{"name":"waitUntil {}","time":{"start":1681299816391,"stop":1681299817562,"duration":1171},"status":"passed","steps":[{"name":"uncaught exception TypeError: ReactDOM.createRoot is not a function","time":{"start":1681299816393,"stop":1681299817562,"duration":1169},"status":"failed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"get [data-e2e=\"APP__CARD\"]","time":{"start":1681299817554,"stop":1681299817554,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"assert expected **undefined** to have a length of **2** but got **0**","time":{"start":1681299817554,"stop":1681299817554,"duration":0},"status":"failed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"[data-e2e=\"APP__CARD\"] request","time":{"start":1681299817554,"stop":1681299817554,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"[data-e2e=\"APP__CARD\"] request","time":{"start":1681299817555,"stop":1681299817555,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"[data-e2e=\"APP__CARD\"] request","time":{"start":1681299817555,"stop":1681299817555,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"[data-e2e=\"APP__CARD\"] request","time":{"start":1681299817555,"stop":1681299817555,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"[data-e2e=\"APP__CARD\"] request","time":{"start":1681299817555,"stop":1681299817555,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"[data-e2e=\"APP__CARD\"] request","time":{"start":1681299817555,"stop":1681299817555,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"[data-e2e=\"APP__CARD\"] request","time":{"start":1681299817555,"stop":1681299817555,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"[data-e2e=\"APP__CARD\"] request","time":{"start":1681299817555,"stop":1681299817555,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"[data-e2e=\"APP__CARD\"] request","time":{"start":1681299817555,"stop":1681299817555,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"[data-e2e=\"APP__CARD\"] request","time":{"start":1681299817555,"stop":1681299817555,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"[data-e2e=\"APP__CARD\"] request","time":{"start":1681299817562,"stop":1681299817562,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"[data-e2e=\"APP__CARD\"] request","time":{"start":1681299817562,"stop":1681299817562,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"[data-e2e=\"APP__CARD\"] request","time":{"start":1681299817562,"stop":1681299817562,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"attachments":[],"parameters":[],"stepsCount":16,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":true}],"attachments":[{"uid":"55eaf3584f326fe5","name":"e2e/tests/runAll.cy.ts:2023-04-12T11:43:37.202Z:attempt-1","source":"55eaf3584f326fe5.png","type":"image/png","size":154144},{"uid":"93401d12fd7074ae","name":"video recording","source":"93401d12fd7074ae.mp4","type":"video/mp4","size":3764693}],"parameters":[],"stepsCount":21,"attachmentsCount":2,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[],"labels":[{"name":"package","value":"e2e.tests.runAll.cy.ts"},{"name":"parentSuite","value":"Vite React Microfrontends"},{"name":"suite","value":"It checks host app"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"1d44adb11d1c8c8d","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1681299814766,"stop":1681299816030,"duration":1264}}],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":27,"broken":0,"skipped":0,"passed":110,"unknown":0,"total":137},"items":[{"uid":"47faa07f7daed94c","reportUrl":"https://module-federation.github.io/module-federation-examples/687//#testresult/47faa07f7daed94c","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1681214582753,"stop":1681214584841,"duration":2088}},{"uid":"72f648222185d237","reportUrl":"https://module-federation.github.io/module-federation-examples/686//#testresult/72f648222185d237","status":"passed","time":{"start":1681146971328,"stop":1681146974419,"duration":3091}},{"uid":"c0c23278f60dee08","reportUrl":"https://module-federation.github.io/module-federation-examples/685//#testresult/c0c23278f60dee08","status":"passed","time":{"start":1681146971328,"stop":1681146974419,"duration":3091}},{"uid":"8fad930781c72515","reportUrl":"https://module-federation.github.io/module-federation-examples/684//#testresult/8fad930781c72515","status":"passed","time":{"start":1681136333380,"stop":1681136335103,"duration":1723}},{"uid":"598eaffa3e8e7dd2","reportUrl":"https://module-federation.github.io/module-federation-examples/683//#testresult/598eaffa3e8e7dd2","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1681099512126,"stop":1681099513830,"duration":1704}},{"uid":"f0898dac8714af35","reportUrl":"https://module-federation.github.io/module-federation-examples/682//#testresult/f0898dac8714af35","status":"passed","time":{"start":1681098141112,"stop":1681098142899,"duration":1787}},{"uid":"a845667b0c926a6c","reportUrl":"https://module-federation.github.io/module-federation-examples/671//#testresult/a845667b0c926a6c","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1680987658955,"stop":1680987660531,"duration":1576}},{"uid":"5a13f30c6188edb","reportUrl":"https://module-federation.github.io/module-federation-examples/665//#testresult/5a13f30c6188edb","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1680909809102,"stop":1680909810483,"duration":1381}},{"uid":"2aa42e6c10abab1c","reportUrl":"https://module-federation.github.io/module-federation-examples/661//#testresult/2aa42e6c10abab1c","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1680834532752,"stop":1680834534469,"duration":1717}},{"uid":"5d04e5e282a54d9d","reportUrl":"https://module-federation.github.io/module-federation-examples/657//#testresult/5d04e5e282a54d9d","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1680806547457,"stop":1680806549202,"duration":1745}},{"uid":"a55323754beb9004","reportUrl":"https://module-federation.github.io/module-federation-examples/630//#testresult/a55323754beb9004","status":"passed","time":{"start":1680739147837,"stop":1680739150032,"duration":2195}},{"uid":"62bdc2eddf09bd78","reportUrl":"https://module-federation.github.io/module-federation-examples/619//#testresult/62bdc2eddf09bd78","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1680701551565,"stop":1680701553845,"duration":2280}},{"uid":"3b1dc7175722d99d","reportUrl":"https://module-federation.github.io/module-federation-examples/616//#testresult/3b1dc7175722d99d","status":"passed","time":{"start":1680695673511,"stop":1680695675221,"duration":1710}},{"uid":"34cc530e8c8a6c61","reportUrl":"https://module-federation.github.io/module-federation-examples/615//#testresult/34cc530e8c8a6c61","status":"passed","time":{"start":1680695673511,"stop":1680695675221,"duration":1710}},{"uid":"25084eb5ebd6712b","reportUrl":"https://module-federation.github.io/module-federation-examples/614//#testresult/25084eb5ebd6712b","status":"passed","time":{"start":1680695673511,"stop":1680695675221,"duration":1710}},{"uid":"2cd9f1b6131b2b72","reportUrl":"https://module-federation.github.io/module-federation-examples/607//#testresult/2cd9f1b6131b2b72","status":"passed","time":{"start":1680613118927,"stop":1680613120549,"duration":1622}},{"uid":"396b24b35468111d","reportUrl":"https://module-federation.github.io/module-federation-examples/602//#testresult/396b24b35468111d","status":"failed","statusDetails":"The following error originated from your application code, not from Cypress. It was caused by an unhandled promise rejection.\n\n  > ReactDOM.createRoot is not a function\n\nWhen Cypress detects uncaught errors originating from your application it will automatically fail the current test.\n\nThis behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.\n\nhttps://on.cypress.io/uncaught-exception-from-application","time":{"start":1680566198632,"stop":1680566200068,"duration":1436}},{"uid":"4cddf4b0abffaf6a","reportUrl":"https://module-federation.github.io/module-federation-examples/597//#testresult/4cddf4b0abffaf6a","status":"passed","time":{"start":1680280131774,"stop":1680280133467,"duration":1693}},{"uid":"da801d4fe631e7d6","reportUrl":"https://module-federation.github.io/module-federation-examples/595//#testresult/da801d4fe631e7d6","status":"passed","time":{"start":1680280131774,"stop":1680280133467,"duration":1693}},{"uid":"c17947516ecf9d4e","reportUrl":"https://module-federation.github.io/module-federation-examples/594//#testresult/c17947516ecf9d4e","status":"passed","time":{"start":1680191673204,"stop":1680191674806,"duration":1602}}]},"tags":[]},"source":"5ce8d0f3fc5b89af.json","parameterValues":[]}